api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -124,18 +137,47 @@ def get_census_version_description(census_version: str) -> CensusVersionDescript
     return description
 
 
-def get_census_version_directory() -> Dict[CensusVersionName, CensusVersionDescription]:
+def get_census_version_directory_of_lts_releases(
+    include_retracted: bool = False,
+) -> Dict[CensusVersionName, CensusVersionDescription]:
     """
-    Get the directory of Census releases currently available.
+    Get a view of the directory that only includes the long-term stable ("LTS") Census releases, optionally including
+    retracted releases.
+
+    Params:
+        include_retracted: Include retracted releases in the returned directory.
 
     Returns:
-        A dictionary that contains release names and their corresponding release description.
+        A dictionary that contains release names and their corresponding release description, for LTS releases only.
 
     Lifecycle:
         maturing
 
     See Also:
+        :func:`get_census_version_directory`: get directory of all Census releases currently available.
         :func:`get_census_version_description`: get description by census_version.
+    """
+    release_flags = dict(lts=True) if include_retracted else dict(lts=True, retracted=False)
+    return get_census_version_directory(**release_flags)
+
+
+# TODO: Update example output and add example for use of release_flags
+def get_census_version_directory(**release_flags_kwargs: bool) -> Dict[CensusVersionName, CensusVersionDescription]:
```

Comment:
need docstring info on flags - or better yet, a pointer to where they are documented.

also need additional examples highlighting most common use cases such as lts=True

Reply:
Since this is intended to be a "mechanism"-only method, I opted to have keep the docstring agnostic about the admissible flags. If we remove, `get_census_version_directory_of_lts_releases()`, then it definitely makes sense to document the flags here.

Reply:
the (large) downside IMHO is the loss of typing information (both IDE hints, linting, etc)

Explicit signatures are much preferred IMHO, and in real-world use no less flexible.

Reply:
As that's what I proposed in the original design changes, I'm not opposed to it. But I thought the outcome of the discussion was to remain agnostic about the specific flags in this method, and in the releases.json file. But maybe just the latter was the case?

Reply:
I agree we wanted flexible flags over the long-term in the data model (release.json). But in any given API release, we are not agnostic and we code for a specific schema.  So the untyped signature doesn't really do much other than disable tooling help

Reply:
Now typed!